com.unity.perception/Runtime/GroundTruth/Labelers/Visualization/HUDPanel.cs

```diff
@@ -13,119 +14,125 @@ namespace UnityEngine.Perception.GroundTruth
     /// </summary>
     public class HUDPanel : MonoBehaviour
     {
-        Dictionary<string, (bool, KeyValuePanel)> entries = new Dictionary<string, (bool, KeyValuePanel)>();
-        Stack<KeyValuePanel> orphans = new Stack<KeyValuePanel>();
+        readonly Dictionary<string, Dictionary<string, string>> entries = new Dictionary<string, Dictionary<string, string>>();
 
-        /// <summary>
-        /// The panel that will hold all of the key value panel elements, this reference is needed to be able to hide the panel
-        /// </summary>
-        public GameObject contentPanel = null;
-        /// <summary>
-        /// The scroll rect of the HUD panel, this reference is needed to be able to hide the panel
-        /// </summary>
-        public ScrollRect scrollRect = null;
-        /// <summary>
-        /// The background image of the HUD panel, this reference is needed to be able to hide the panel
-        /// </summary>
-        public Image img = null;
+        private GUIStyle keyStyle;
+        private GUIStyle valueStyle;
 
-        void Update()
+        private const int LineHeight = 22;
```

Comment:
Private should be omitted where possible, and non-public const members should start with k_. It would be good to follow the steps here https://github.com/Unity-Technologies/com.unity.perception#suggested-ide-setup and fix the warnings it shows.
```suggestion
        const int k_LineHeight = 22;
```

Reply:
Updated to match coding style guidelines.